components/service/glean/src/main/java/mozilla/components/service/glean/Glean.kt

```diff
@@ -233,6 +238,9 @@ open class GleanInternalAPI {
             throw AssertionError("Could not get own package info, aborting init")
         }
 
+        // Create a MetricsPing object
```

Comment:
nit: it's MetricsPingScheduler now, but the comment doesn't add much to the code, so you can remove it I guess :)